vmaas/types.go

```diff
@@ -55,6 +55,69 @@ type CvesRequest struct {
 	PageSize            int        `json:"page_size"`
 }
 
+type StringSlice []string
+
+func (t *StringSlice) UnmarshalJSON(data []byte) error {
+	if string(data) == "null" || string(data) == `""` || len(data) == 0 {
+		return nil
+	}
+	if data[0] == '[' {
+		var val []string
+		err := json.Unmarshal(data, &val)
+		if err != nil {
+			return err
+		}
+		*t = val
+		return nil
+	}
+	if data[0] == '"' {
+		var val string
+		err := json.Unmarshal(data, &val)
+		if err != nil {
+			return err
+		}
+		*t = []string{val}
+		return nil
+	}
+	return errors.New("failed to unmarshall StringSlice")
+}
+
+func (t *StringSlice) MarshalJSON() ([]byte, error) {
+	if *t == nil {
+		return []byte("null"), nil
+	}
+	if len(*t) == 0 {
+		return []byte("[]"), nil
+	}
+
+	res := []byte{'['}
+	for _, item := range *t {
+		if item == "" {
+			res = append(res, []byte("null,")...)
+			continue
+		}
+		val, err := json.Marshal(item)
+		if err != nil {
+			return nil, err
+		}
+		res = append(res, val...)
+		res = append(res, ',')
+	}
+
+	res[len(res)-1] = ']' // replace the trailing comma with the closing ]
+	return res, nil
+}
+
+type ErrataRequest struct {
+	ErrataList    []string    `json:"errata_list"`
```

Comment:
nitpick, we know it is a list
```suggestion
	Errata    []string    `json:"errata_list"`
```